lib/usd/utils/tests/test_MayaTransformAPI.cpp

```diff
@@ -0,0 +1,1196 @@
+#include <gtest/gtest.h>
```

Comment:
Thanks for the nice test.  Did not look at this line by line, just looked at the test intent for each routine, which seems to cover things.  Hopefully a code coverage tool would say the same!